src/login/translation/languages/en.json

```diff
@@ -106,6 +106,8 @@
   "emails.code_invalid": "The confirmation code is invalid, please try again or request a new code",
   "emails.code_invalid_or_expired": "The confirmation code is invalid or has expired, please try again or request a new code",
   "emails.confirm_email_placeholder": "Email confirmation code",
+  "emails.invalid_code": "Invalid confirmation code, only 32 characters and 4 hyphens allowed",
+  "emails.confirm_code_wrong_length": "This code is too long, the code to confirm your email addresses must be 36 characters long",
```

Comment:
I would change "_your email address**es**_" to "_your email address_" as you only confirm one address at a time.